config/lighthouse.php

```diff
@@ -31,6 +31,8 @@
         'middleware' => [
             \Nuwave\Lighthouse\Support\Http\Middleware\AcceptJson::class,
 
+            \Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful::class,
```

Comment:
I have this as first middleware. Not sure whether it matters.

Reply:
Also, a couple lines below should be `'guard' => 'sanctum'` (I could not annotate the line directly because it wasn't changed):

https://github.com/nuwave/lighthouse-example/blob/2e7ade3bec574290fdbce8f498d663bfaea6fa89/config/lighthouse.php#L49-L59

Reply:
> I have this as first middleware. Not sure whether it matters.

The `AcceptJson` middleware will cause errors during `EnsureFrontendRequestsAreStateful` to be returned as JSON, which plays better with GraphQL Playground.